src/main/java/Homework2/Random_Chars_Table/RandomCharsTable.java

```diff
@@ -0,0 +1,84 @@
+package Homework2.Random_Chars_Table;
+
+import java.util.Scanner;
+
+public class RandomCharsTable {
+    int length = 0;
+    int height = 0;
+    String input = "";
+
+    static char getRandomChar(int min, int max) {
+        max -= min;
+        char randomCh = (char) ((int) (Math.random() * ++max) + min);
+        return randomCh;
```

Comment:
this method is too confusing and not correct.
1) max - min = delta (not max!!!)
2) (Math.random() * ++max - Why the incrimination?